tests/e2e/specs/BoundarySignalEvent.spec.js

```diff
@@ -0,0 +1,54 @@
+import {
+  dragFromSourceToDest,
+  getElementAtPosition,
+  moveElement,
+  removeIndentationAndLinebreaks,
+  setBoundaryEvent,
+  typeIntoTextInput,
+  waitToRenderAllShapes,
+} from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+describe('Boundary Signal Event', () => {
+  const taskPosition = { x: 200, y: 200 };
+  const boundarySignalEventPosition = { x: 260, y: 200 };
+
+  beforeEach(() => {
+    dragFromSourceToDest(nodeTypes.task, taskPosition);
+    setBoundaryEvent(nodeTypes.boundarySignalEvent, taskPosition);
+  });
+
+  it('update boundary signal event properties element', () => {
+    moveElement(taskPosition, boundarySignalEventPosition.x, boundarySignalEventPosition.y);
+
+    const name = 'Test name';
+    typeIntoTextInput('[name=name]', name);
+
+    const signalXML = '<bpmn:boundaryEvent id="node_3" name="Test name" attachedToRef="node_2"><bpmn:signalEventDefinition /></bpmn:boundaryEvent>';
```

Comment:
You can re-use the `name` variable here:
```suggestion
    const signalXML = `<bpmn:boundaryEvent id="node_3" name="${name}" attachedToRef="node_2"><bpmn:signalEventDefinition /></bpmn:boundaryEvent>`;
```